### DISCUSSION TITLE
RFC: Backport candidates

### DISCUSSION THREAD

**Discussion by effigies** [nipreps/fmriprep#2908] [Development]
I've created a [![backport candidate](https://img.shields.io/github/labels/nipreps/fmriprep/backport%20candidate)](https://github.com/nipreps/fmriprep/pulls?q=is%3Apr+is%3Aopen+label%3A%22backport+candidate%22) label.

With the best of intentions with maintenance branches and bug-fix protocols, we still generally fork fixes off of `master` and fixes that could go into an older maintained series do not. The purpose of this label is to mark PRs that are targeting `master` (or a newer `maint/*` branch) as fixes that could go in earlier maintenance branches, if and when a new release is called for.

This way we can keep our usual process without forcing rebases of fixes onto maintenance branches, merging, and then merging the maintenance branch into `master`. When it's time to release a bug-fix release, we can check the label for any relevant PRs, perform the backports in a single PR.

Thoughts?

**Comment by mgxd** [comment]
I like it - labels are findable in the github api (https://api.github.com/repos/nipreps/fmriprep/pulls/2900) so we could eventually automate this as well.
